extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -456,13 +458,16 @@ export class AmpLightboxGallery extends AMP.BaseElement {
         this.measureMutateElement(measureOverflowState, mutateOverflowState);
       });
     }
+    triggerAnalyticsEvent(this.element, 'descriptionToggled', {});
```

Comment:
What is description toggled supposed to mean? `updateDescriptionBox` gets called every time you swipe to a new slide in order to update the contents of the description. It isn't a user action. The user can tap once to bring up or make all the controls disappear (gallery/close buttons, left/right arrows, AND the description). You can fire that in `onToggleControls_`, but I think this event doesn't really make sense. 
Propose to either: 1. remove this event, or 2. expose 'controlsToggled' instead.

Reply:
Let's expose `controlsToggled` cause I think that is worth tracking. Made the change in the latest commit.